.github/workflows/ci-proof-pack.yml

```
@@ -0,0 +1,166 @@
+                echo "$f\t$t\tYES" >> scan.tsv
+                echo "FILE=$f TOKEN=$t FORBIDDEN_HIT=YES" >> ci_proof.txt
+                HIT_ANY="1"
+              else
+                echo "$f\t$t\tNO" >> scan.tsv
+                echo "FILE=$f TOKEN=$t FORBIDDEN_HIT=NO" >> ci_proof.txt
+              fi
+            done
+          done < changed_files.txt
+
+          python3 - << 'PY'
+import json, os, datetime
```

actionlint (1.7.10)

could not parse as YAML: could not find expected ':'

(syntax-check)